components/gallery/ModalGallery.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useEffect } from "react";
+import ReactDOM from "react-dom";
+import HorizontalGallery from "./horizontalScrollGallery";
+import { Position } from "@cloudinary/url-gen/qualifiers";
+
+const Modal = ({ onClose, public_id }) => {
```

Comment:
@jerwin21 Added public_id as a prop and remove hardcoded public id to `HorizontalGallery`